cluster/images/hyperkube/Makefile

```diff
@@ -1,6 +1,9 @@
-# build the hyperkube image.
+# Build the hyperkube image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-VERSION=v1.1.2
+REGISTRY?="gcr.io/google_containers"
```

Comment:
@mikedanese Do you mean that we should `source version.sh` and get the default version from there? Or should we go with user-defined version flags?